src/components/ProviderList.js

```diff
@@ -17,7 +18,7 @@ export default function ProviderList({ providerList }) {
         </a>
       </ListHeader>
       {flatrateProviders ? (
-        <StyledList role="list">
+        <List role="list">
           {flatrateProviders?.map(
```

Comment:
Don't think you need the optional chaining here 😉

Reply:
and here too